central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/editors/xpl/ConnectorDescriptorItemUi.java

```diff
@@ -171,7 +171,6 @@ public ConnectorDescriptorItemUi(DiscoveryViewer discoveryViewer,
 		// help UI tests
 		checkbox.setData("connectorId", connector.getId()); //$NON-NLS-1$
 		checkbox.setVisible(connector.isInstallable());
-		checkbox.setBackground(background);
```

Comment:
Nice, that seems to be the line that was actually causing the invisible checkbox issue.